DopplerCustomDomain/Api/CustomDomainController.cs

```diff
@@ -58,6 +60,17 @@ public async Task<IActionResult> CreateCustomDomain([FromRoute] string domainNam
                 return new NotFoundObjectResult($"Cannot find the service called: {domainConfiguration.service}");
             }
 
+            var ipResolutionValidationSuccess = await _dnsResolutionValidator.IsNamePointingToOurServiceAsync(domainName);
+
+            if (!ipResolutionValidationSuccess && domainConfiguration.ipResolutionValidation == IPResolutionValidation.Required)
+            {
+                return new BadRequestObjectResult($"{domainName} does not resolve to our service IP address");
+            }
+            else if (!ipResolutionValidationSuccess && domainConfiguration.ipResolutionValidation != IPResolutionValidation.Required)
```

Comment:
The log message says `Ignored`, I think that is better to use the same value of the `IPResolutionValidation` enum, and will continue working as expected if, in the future, we add another possible value to `IPResolutionValidation`
```suggestion
            else if (!ipResolutionValidationSuccess && domainConfiguration.ipResolutionValidation == IPResolutionValidation.Ignored)
```